docs/project_rerum_admin_work/about.md

```diff
@@ -0,0 +1,113 @@
+
+- **Current Tech Lead:** Lalith Adithya Reddy Avuthu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/alar12)  
+
+- **Developers:**
+    - Bryan Haberberger [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/thehabes)
+    - Patrick Cuba [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/cubap)
+
+- **Start Date:**  March 2023
+- **Adoption Date:**  Aug 2024
+- **Technologies Used:**
+    - Tech Stack: 
+                  - Web development (Html, CSS, JavaScript), RESTful API Development (RERUM APIs)
+                  - Node.js & JavaScript
+                  - CI/CD Pipelines
+- **Type:** Web application
+- **License:** Primary Developers: Bryan Haberberger and Patrick Cuba © 2023 Research Computing Group at Saint Louis University
+
+### User Guide
+
+#### TinyNode
+
+1. **Home Page**:
+   - When you open the TinyNode web application, you will be directed to the Home page. This page provides an introduction to the application.
+
+2. **Interacting with the API**:
+   - Navigate through the interface to interact with the RERUM API.
+   - You can perform basic API calls, retrieve data, and manage it directly through the application.
+
+3. **Viewing Data**:
+   - The application allows you to fetch data from the RERUM API.
+   - After fetching, you can view and manage this data within the app.
+
+4. **Development and Testing**:
+   - Developers can use the application to test their API integrations and see how data management works. 
+
+
+## Technical Information
+
+[![Architecture](https://mermaid.ink/img/pako:eNqNUsty2jAU_RWNsuVly4bgRWewTBo6JcNgsqnIQrGvwcVIHklOoAz_XoOgNZnpQ6v7OvecI-mAE5kCDvBK8XKNFtFSoPro6tUWHpQUpj0Wqa2fzshhj4vp1y6N4-6XGE2EAZXxBF5Qu_0JjVz2rEHZMk9MLsVLA-vaIcJGswmivCj0pQtXig8CQp5sbvlDhz3VmjvfNYpBvYGyxKHLxrtSgdanzoPiW3iXatMgDy15aMnnsjKgm21i2x6bj-fPU3QaOrmodXxwEXp20mcRNxxNueAr2IIw__BCJ10a_V5DHfY5N4_VKxqd70lbI9RldIJmeQlFLqBBS61-Sti8EmgB2jTlUyufeiyCspB7ZCSK4A0KWTaknSetfOqzmZJpdeZGFvRnE6PL7Tg2Df3LQ9qU-jfdi2-zL6D-LijLiyK4y4ZZSxslNxDcEUIucfs9T806cMtdExReQcOs_98gegX1k78z4RbegtryPK0__uG0YonNun7BJQ7qMOVqs8RLcazneGVkvBcJDoyqoIWVrFZrHGS80HVWlSk3EOW8ft_tr2rJxTcpb3IcHPAOB21y3-mTQZ_0ie8Rz63DFt7jwHG9zr3TGw6cnt8bDIkzOLbwj_MO5_gTGNkQFA?type=png)](https://mermaid.live/edit#pako:eNqNUsty2jAU_RWNsuVly4bgRWewTBo6JcNgsqnIQrGvwcVIHklOoAz_XoOgNZnpQ6v7OvecI-mAE5kCDvBK8XKNFtFSoPro6tUWHpQUpj0Wqa2fzshhj4vp1y6N4-6XGE2EAZXxBF5Qu_0JjVz2rEHZMk9MLsVLA-vaIcJGswmivCj0pQtXig8CQp5sbvlDhz3VmjvfNYpBvYGyxKHLxrtSgdanzoPiW3iXatMgDy15aMnnsjKgm21i2x6bj-fPU3QaOrmodXxwEXp20mcRNxxNueAr2IIw__BCJ10a_V5DHfY5N4_VKxqd70lbI9RldIJmeQlFLqBBS61-Sti8EmgB2jTlUyufeiyCspB7ZCSK4A0KWTaknSetfOqzmZJpdeZGFvRnE6PL7Tg2Df3LQ9qU-jfdi2-zL6D-LijLiyK4y4ZZSxslNxDcEUIucfs9T806cMtdExReQcOs_98gegX1k78z4RbegtryPK0__uG0YonNun7BJQ7qMOVqs8RLcazneGVkvBcJDoyqoIWVrFZrHGS80HVWlSk3EOW8ft_tr2rJxTcpb3IcHPAOB21y3-mTQZ_0ie8Rz63DFt7jwHG9zr3TGw6cnt8bDIkzOLbwj_MO5_gTGNkQFA)
+
+### **Architecture**
```

Comment:
Please restructure this as a paragraph of text under a "Technical Overview" section to match the structure of other existing pages. See https://oss-slu.github.io/docs/project_pi4micronaut/about and https://oss-slu.github.io/docs/project_wheres_religion_mobile/about for examples of the sort of format. Consistency in this format is important for branding.

Reply:
to be clear, the mermaid diagram is fine. i'm talking about the multiple bulleted lists from lines 66-91.